lib/langchain/vectorsearch/elasticsearch.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+module Langchain::Vectorsearch
+  class Elasticsearch < Base
```

Comment:
Could you please add code comments? See the rest of the similar `/vectorsearch/*` classes for examples.